.github/workflows/pull-request-integration.yml

```diff
@@ -0,0 +1,67 @@
+name: pull-request-integration
+
+on:
+  pull_request_target:
+    branches: [ main ]
+
+env:
+  NAMESPACE: community
+  COLLECTION_NAME: digitalocean
+
+concurrency:
+  group: cloud-integration-tests
+  cancel-in-progress: false
+
+jobs:
+
+  integration:
+    # Require reviewers for this environment
+    environment: integration
+
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
+
+    steps:
+
+      # https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          path: ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
+          ref: ${{ github.event.pull_request.head.sha }} # Check out the pull request
+          persist-credentials: false
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
```

Comment:
Matrix is not defined anywhere, so this will probably fail to "compile".

Reply:
Ah yeah, bad copy pasta, I'll fix.